vault/resource_ssh_secret_backend_role.go

```diff
@@ -183,6 +183,11 @@ func sshSecretBackendRoleResource() *schema.Resource {
 			Optional:    true,
 			Computed:    true,
 		},
+		"allow_empty_principals": {
+			Type:     schema.TypeBool,
+			Optional: true,
+			Default:  false,
```

Comment:
Looks like Vault enforces a [default](https://developer.hashicorp.com/vault/api-docs/secret/ssh#allow_empty_principals) for this field. Should we make this Computed and let Vault set the default?

Reply:
Never mind, I see your comment below that this can not be read from the API. I wonder if we should make a note to update the Vault docs in that case (not a blocker for this PR though)

Reply:
What update to the Vault docs did you have in mind?

Reply:
Sorry, I initially thought that the docs showed this field as being returned, but I had misread it for a different field. I think we should be okay 🙇🏼